Code/Tools/AzTestRunner/src/main.cpp

```diff
@@ -126,6 +126,14 @@ namespace AzTestRunner
                 std::cout << "arg[" << i << "] " << argv[i] << std::endl;
             }
 
+            // Construct a full retry command
+            std::cout << "Full command: " << argv[0] << " " << lib << " " << symbol;
+            for (int i = 1; i < argc; i++)
+            {
+                std::cout << " " << argv[i];
+            }
+            std::cout << std::endl;
```

Comment:
isnt this printing the same thing that is being printed in lines 124-127 just with a different format?
If so, why do you need this in that format?

Reply:
It's more or less the same information. This is constructing an easy copy/paste command to rerun the same test.

Reply:
I don't think it should output this "easy to rerun" format at the start of every execution. It should output it at the end, only if a failure occurs, and preferably in the same way as in LyTT.  That would be: a CLI command for only rerunning the failed test(s), including the full path to AzTestRunner and its arguments.

For reference: o3de\Tools\LyTestTools\ly_test_tools\_internal\pytest_plugin\terminal_report.py

Reply:
Moved retry command to after test fails